drake/systems/plants/RigidBodySystem.h

```diff
@@ -224,12 +229,20 @@ class DRAKERBSYSTEM_EXPORT RigidBodySystem {
   friend DRAKERBSYSTEM_EXPORT StateVector<double> getInitialState(
       const RigidBodySystem& sys);
 
+  /*!
+   * Returns a string describing the semantics of the state vector.
+   */
+  std::string getStateVectorSemantics() const;
+
   // some parameters defining the contact
   bool use_multi_contact;
   double penetration_stiffness;  // k
   double penetration_damping;    // b
   double friction_coefficient;   // mu
 
+  // The name of the file used to instantiate this rigid body system
+  std::string filename;
```

Comment:
there may be multiple files (e.g. if I called addRobotFromURDF() multiple times).